Vizzuality/landgriffon
# (feat) LANDGRIF-1279 check scenario permissions for comparisons

### General description

Checks auth over scenarios when it comes to comparison request, that is, tables and charts

Due to a handicap in the FE, this needs to be also checked in the API

### Designs

_Link to the related design prototypes (if applicable)_

### Testing instructions

[These should be](https://vizzuality.atlassian.net/browse/LANDGRIF-1279) the use cases in the scope of this PR

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
